chiselFrontend/src/main/scala/Chisel/When.scala

```diff
@@ -2,9 +2,12 @@
 
 package Chisel
```

Comment:
Yeah, the .otherwise transform can do away. But just in case, I prefer to have anything that returns not Unit have a transform.